h2o-algos/src/main/java/hex/adaboost/AdaBoostModel.java

```diff
@@ -10,7 +10,7 @@
 public class AdaBoostModel extends Model<AdaBoostModel, AdaBoostModel.AdaBoostParameters, AdaBoostModel.AdaBoostOutput> {
     private static final Logger LOG = Logger.getLogger(AdaBoostModel.class);
 
-    public enum Algorithm {DRF, GLM, GBM, AUTO}
+    public enum Algorithm {DRF, GLM, GBM, DEEP_LEARNING,AUTO}
```

Comment:
Shouldn't we use pascal case for `DeepLearning`? I haven't found this snake case in h2o repo. I would try to be consistent with StackedEnsambles or AutoML.

Reply:
Found it, I was looking to Rulefit, where it is Java style: 

https://github.com/h2oai/h2o-3/blob/9f811a7d1319612c32eb37cce3222d7def5d8143/h2o-algos/src/main/java/hex/rulefit/RuleFitModel.java#L21

https://github.com/h2oai/h2o-3/blob/9f811a7d1319612c32eb37cce3222d7def5d8143/h2o-algos/src/main/java/hex/rulefit/RuleFitModel.java#L23

What do you think? I can refactor to AutoML, SE, Infogram style.

Reply:
We have inconsistencies also here: https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/rulefit.html#common-parameters

Reply:
It will be disused and solved in different PR.

Reply:
Yes, we have inconsistencies in our naming structure.  However since we call deeplearning as one word in the algos, let's keep it consistent with the original name and use DeepLearning instead of deep_learning.